tests/unit/postTestBuildComment.ts

```diff
@@ -1,7 +1,9 @@
 import * as core from '@actions/core';
+import type {RestEndpointMethods} from '@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types';
 import {when} from 'jest-when';
 import asMutable from '@src/types/utils/asMutable';
 import ghAction from '../../.github/actions/javascript/postTestBuildComment/postTestBuildComment';
+import type {CreateCommentResponse} from '../../.github/libs/GithubUtils';
 import GithubUtils from '../../.github/libs/GithubUtils';
```

Comment:
You can use absolute imports
```suggestion
import ghAction from '@github/actions/javascript/postTestBuildComment/postTestBuildComment';
import type {CreateCommentResponse} from '@github/libs/GithubUtils';
import GithubUtils from '@github/libs/GithubUtils';
```